.travis.yml

```diff
@@ -65,12 +64,10 @@ matrix:
       env: PROJECT_DIR=examples/CURL TOOLCHAIN=clang-libstdcxx
     - os: osx
       env: PROJECT_DIR=examples/CURL TOOLCHAIN=osx-10-9
```

Comment:
Travis images for OSX changed recently, defaults now osx-10-11 and ios-9-3. I've pushed update of configuration, please rebase this one too.